cage_player.cpp

```diff
@@ -0,0 +1,27 @@
+#include "backend/player_command.hpp"
+#include "natives.hpp"
+#include "pointers.hpp"
+#include "util/ped.hpp"
+#include "util/entity.hpp"
+
+namespace big
+{
+	class cage_player : player_command
+	{
+		using player_command::player_command;
+
+		virtual void execute(player_ptr player, const std::vector<std::uint64_t>& _args, const std::shared_ptr<command_context> ctx)
+		{
+			
+			
+			
```

Comment:
very werid pr here with lots of unneeded spaces as seen where this comment is, duplicated files set in the src, entire pr should be fixed,